### ISSUE TITLE
Mapping coordinates: Pinto

### ISSUE THREAD

**Issue by damianooldoni** [inbo/sovon#105]
@EricStienen: 
> De coordinaten van de oude data worden niet juist geplot! Pinto in Spanje ligt volgens de SOVON-kaartjes bijvoorbeeld ergens in de Middelandse Zee in de buurt van Mallorca.

@EricStienen: I checked this observation and it seems an error in the INBO database:
Latitdue: 40°14'0''
Longitude: 3°40'0''	
I converted latitude/longitude, expressed in degrees, minutes and seconds to decimal latitude/longitude (6.2.6 and 6.2.7: https://inbo.github.io/cr-birding/4_observations.html#observation-latitude). You can do the same with online tools, for example: https://www.fcc.gov/media/radio/dms-decimal
Same result returned. So, the mapping is correct. 
If you use the result on Google Maps or OpenStreetMaps you get indeed a point near to Mallorca. I think the person who put the observation in database confused the city Pinto (near to Madrid) with Isla del Pinto. If you search for Pinto on Google Maps, you get the island as first result: https://www.google.be/maps/search/Pinto/@40.1430311,0.9918663,8z/data=!3m1!4b1.

@EricStienen : I see there are a lot of data linked to Pinto. Give me please the right coordinates and I will change them easily in my mapping script. 

@EricStienen : if you find other problems with coordinates, please let me know.

**Comment by EricStienen** [comment]
@damianooldoni   : all 861 coordinates coordinates in our DB of Pinto are Latitude: 40°14'0'' N
 and Longitude: 3°40'0'' W (in fact slightly wrong they should be Latitude: 40°15'41''N and Longitude: 3°38'02'' W - see google earth map below). But that slight deviation is is not the problem.

![image](https://user-images.githubusercontent.com/6673251/58867303-7f57a480-86ba-11e9-809b-a27ddd56bc51.png)

**Comment by EricStienen** [comment]
The problem is that if you translate this to decimal degrees you derive at about 40.26 and -3.64).  I think that somebody did not account for EAST and WEST. 

In other words probably all coordinates with W are not plotted in the right way!

**Comment by EricStienen** [comment]
Same accounts for N and S, but we still do not have recoveries below the equator.

**Comment by damianooldoni** [comment]
Thanks @EricStienen to check this! I will have a look asap.

**Comment by damianooldoni** [comment]
You are right. I converted to decimal but didn't think that the direction was coded in separate columns, `PlaatsLengtegraadRichtingCode` and `PlaatsBreedtegraadRichtingCode`. Thanks for checking. And sorry for the mistake.

@JeroenNienhuis: I made a file in attachment with all our observations where the longitude should be negative. The file contains two columns: `observation_reference` and `observation_location`. We speak about 11187 observations linked to 875 distinct locations. Would it be possible to you to apply changes automatically in db?

[observation_reference_with_negative_longitude.txt](https://github.com/inbo/cr-birding/files/3265048/observation_reference_with_negative_longitude.txt)

**Comment by JeroenNienhuis** [comment]
It shoult be correct now. See for example https://submit.cr-birding.org/animals/29657/  (N.HAD)

**Comment by damianooldoni** [comment]
Thanks @JeroenNienhuis. Once again, sorry for the bug. The new data will be with right coordinates.
@EricStienen : can you double check that everything is ok now with coordinates?

**Comment by damianooldoni** [comment]
This problem has been tackled in the new data mapping.

**Comment by damianooldoni** [comment]
It seems there are still some observations with this bug as reported by an observer:

> Hi Eric and Marc,
today I checked some LBBG-data at https://submit.cr-birding.org/
I think there went something wrong with the locations, regions and coordinates in the database (see attached list).
All observations were from Portugal, not from Italy, Tunisia, Algeria or the Mediterranean Sea.

Anyway, this bug has been corrected in the new mapping and the new export of next Wednesady, Oct 7 will hopefully solve this definitely.
